### DISCUSSION TITLE
Homebrew bottle

### DISCUSSION THREAD

**Discussion by austek** [apache/pulsar#20633] [Q&A]
Hi, I'm trying to update the HomeBrew apache-pulsar formula to 3.0.0 https://github.com/Homebrew/homebrew-core/pull/133460/files, Pulsar cli arguments seem to have changed since 2.10.3
2.10.3
standalone was using bin/"pulsar", "standalone", "--zookeeper-dir", "#{testpath}/zk", " --bookkeeper-dir", "#{testpath}/bk" and
for tests #{bin}/pulsar initialize-cluster-metadata -c a -cs localhost -uw localhost -zk localhost
I've changed for 3.0.0
standalone bin/"pulsar", "standalone", "--metadata-dir", "#{testpath}/metadata", " --bookkeeper-dir", "#{testpath}/bk" but the initialize-cluster-metadata is failing https://github.com/Homebrew/homebrew-core/actions/runs/5265568983/jobs/9518538264?pr=133460
Any ideas?

**Comment by tisonkun** [comment]
cc @nicoloboschi I remember that you prepare this formula?
